frontend/src/pages/following-page/styles.module.scss

```diff
@@ -2,8 +2,12 @@
 
 .page-container {
   max-height: 100%;
-  padding: 20px 40px;
+  padding: 20px;
```

Comment:
if you changed the padding on the following page, please insert the same padding on the history and brows page so that they look the same